libraries/chain/apply_context.cpp

```diff
@@ -303,10 +303,19 @@ void apply_context::execute_deferred( deferred_transaction&& trx ) {
    } FC_CAPTURE_AND_RETHROW((trx));
 }
 
-void apply_context::cancel_deferred( uint128_t sender_id ) {
+void apply_context::cancel_deferred( const uint128_t& sender_id ) {
    results.deferred_transaction_requests.push_back(deferred_reference(receiver, sender_id));
 }
-
+/*
+void apply_context::cancel_deferred( const transaction_id_type& trx_id ) {
```

Comment:
If the transaction_id_type's _hash is going to be used as the sender_id, remove this commented out code.